fluentui_controls/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
remove the xmlns:android its not used

Reply:
This is in accordance with other modules. Do you want me to remove it?

Reply:
done